backend/prompt_studio/prompt_studio_output_manager/serializers.py

```diff
@@ -1,9 +1,28 @@
+import logging
+
+from usage.helper import UsageHelper
+
 from backend.serializers import AuditSerializer
 
 from .models import PromptStudioOutputManager
 
+logger = logging.getLogger(__name__)
+
 
 class PromptStudioOutputSerializer(AuditSerializer):
     class Meta:
         model = PromptStudioOutputManager
         fields = "__all__"
+
+    def to_representation(self, instance):
+        data = super().to_representation(instance)
+        try:
+            token_usage = UsageHelper.get_aggregated_token_count(instance.run_id)
+        except Exception as e:
+            logger.error(
+                "Error occured while fetching token usage for run_id"
+                f"{instance.run_id}: {e}"
+            )
+            token_usage = {}
+        data["token_usage"] = token_usage
```

Comment:
@tahierhussain Can you make this a constant if possible?

Reply:
@Deepak-Kesavan Thanks for the catch. Done.